tests/integration/sheet.js

```diff
@@ -255,14 +259,14 @@ describe('Integration: sheet', () => {
           a: {color: 'red'}
         })
         expect(sheet.toString()).to.be(
-          `@media print {\n` +
+          '@media print {\n' +
           `  .${id} {\n` +
-          `    float: left;\n` +
-          `  }\n` +
-          `}\n` +
+          '    float: left;\n' +
+          '  }\n' +
+          '}\n' +
           `.${id} {\n` +
-          `  color: red;\n` +
-          `}`
+          '  color: red;\n' +
+          '}'
```

Comment:
you may be interested in [`stripindent`](https://www.npmjs.com/package/common-tags#stripindent) literal tag from ['common-tags`](https://www.npmjs.com/package/common-tags) project